pennylane/plugins/default_qubit.py

```diff
@@ -240,10 +240,38 @@ def reset(self):
         self._state[0] = 1
         self._pre_rotated_state = self._state
 
-    def probability(self, wires=None):
+    def analytic_probability(self, wires=None):
+        """Return the (marginal) analytic probability of each computational basis."""
         if self._state is None:
             return None
 
         wires = wires or range(self.num_wires)
+
         prob = self.marginal_prob(np.abs(self._state) ** 2, wires)
         return prob
+
+    def _estimate_probability(self, wires=None):
```

Comment:
Now that I think about it more, I'm not sure why I wrote `_estimate_probability()` as private, there is no reason it shouldn't be public as `estimate_probability()`!

This is minor though, and can be addressed in the next PR that modifies `QubitDevice`.

Reply:
I changed it here! Makes sense to have it public (as the other methods). :slightly_smiling_face: